packages/fxa-settings/src/styles/ctas.scss

```diff
@@ -4,7 +4,7 @@
 
 .cta-neutral,
 .cta-primary {
-  @apply py-2 px-5 rounded w-full inline-block text-center border font-header box-border;
+  @apply py-2 px-5 rounded w-full inline-block text-center border font-header box-border truncate transition-standard;
```

Comment:
I think moving `transition-standard` here is a good call, but do we want to add `truncate`? I'm not sure we want an ellipsis in this scenario, might be better to break the CTA text onto two lines even if it's not pretty so the user can read what they're doing. I dunno though, I'd say just leave that out for now unless we have a scenario where we need it already?

Reply:
Oh I can remove truncate yes!